### PR TITLE
Restructure to split ACM downstream deployment connected/disconnected

### PR DESCRIPTION
**Pull request by borball**
Restructure the doc to split the ACM downstream deployment connected/disconnected to make it clear for a beginner.

### CODE REVIEW

**Review by jparrill — CHANGES_REQUESTED** [review]
Apart of the changes requested, LGTM @borball++

File: docs/connected-ZTP-flow-hub-deployment.md [hunk 1]
```diff
@@ -15,25 +15,26 @@
...
 [comment]: <> (TODO: Update ztp-flow-connected.png, specifically the Manifests for Spoke box, with missing arrows to KlusterAddonConfig and ManagedCluster)
 ![](/assets/ztp-flow-connected.png)
 
-But first we need to deploy the OpenShift Hub:
+First we need to deploy the OpenShift Hub:
 
-- if you already did this step, continue on [ACM Deployment](#acm-deployment)
-- If not, please go here and follow the [instructions for the OpenShift Hub Cluster](/docs/prerequirements/ocp4-ipi-deployment.md)
+- if you already did this step, continue on [ACM Deployment Connected](#acm-deployment-connected)
+- If not, please go here and follow the [instructions for the OpenShift Hub Cluster](/docs/prerequirements/ocp4-ipi-deployment.md), then continue on [ACM Deployment Connected](#acm-deployment-connected)
```
[the remark this answers is not in the document]
Comment (borball) [reply docs/connected-ZTP-flow-hub-deployment.md:26]: I originally had 2 lines in both paths, sounds good for me.

File: docs/disconnected-ZTP-flow-hub-deployment.md [hunk 2]
```diff
@@ -50,18 +50,20 @@ After going through these steps, we can continue with the typical flow.
...
 
 ---
 
 ## ACM Deployment in a disconnected Environment
 
-go to the OpenShift Marketplace and look for the "Red Hat Advance Cluster Management" operator and the deploy will start. It takes a while to finish, so please be patience.
+Go to the OpenShift Marketplace and look for the "Red Hat Advance Cluster Management" operator and the deploy will start. It takes a while to finish, so please be patience.
 
-**NOTE**: If you are from QE, DEV or any Red Hat Associate that wanna work with Downstreams versions you need to ask for permissions for this kind of images in the Slack Channel `#forum-acm`. If you already have permissions to do this, you will need to do some extra steps **[explained here](/docs/prerequirements/acm-downstream-deployment.md)**.
+**NOTE**: If you are from QE, DEV or any Red Hat Associate that want to work with Downstream versions, follow steps below to deploy ACM downstream version:
```
[the remark this answers is not in the document]
Comment (borball) [reply docs/disconnected-ZTP-flow-hub-deployment.md:62]: Make sense.

File: docs/connected-ZTP-flow-hub-deployment.md [hunk 3]
```diff
@@ -15,25 +15,26 @@
 
 # Connected ZTP Flow Hub deployment
```
[the remark this answers is not in the document]
Comment (borball) [reply docs/connected-ZTP-flow-hub-deployment.md:17]: Agreed.

**Review by jparrill — APPROVED** [review]

**Pull request merged**